.github/workflows/auto-update-versions.yml

```diff
@@ -9,6 +9,8 @@ on:
 jobs:
   auto-update-versions:
     name: Auto-update versions
+    # restrict forks from running this workflow
+    if: github.repository == 'open-telemetry/opentelemetry.io'
```

Comment:
my lack of yaml-foo -- should the following blocks of code also be indented, or no?

Reply:
job execution conditions are at the `jobs.<job-id>.if` level so this is proper

https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution